site/docs/themes/design-tokens/how-to-read-tokens.mdx

```diff
@@ -0,0 +1,126 @@
+---
+title: How to read semantic tokens
+layout: DetailTechnical
+sidebar:
+  priority: 3
+---
+
+Understanding semantic characteristics means learning how to read token names, so you can easily find and use the right token in both design and code.
+
+<Callout status="warning" title="Don't">
+  Don’t choose a token just because its color looks right—this can cause issues
+  in other themes or modes. Instead, design semantically. Rather than coloring
+  something red, identify it as being in an ‘error’ state, needing ‘attention’,
+  or a ‘negative’ change or message.
+</Callout>
+
+All semantic characteristic tokens in the Salt Design System follow this format:
+
+<Image
+  src="/img/theming/token-naming-structure-light.png"
+  srcDark="/img/theming/token-naming-structure-dark.png"
+  alt=""
+/>
+
+1. **Design system prefix:** A mandatory term that clearly attributes the token to Salt
+2. **Characteristic:** A mandatory term included in all semantic tokens in order to group tokens that share the same purpose or function (e.g., accent, actionable, container, content)
+3. **Modifier:** An optional term that reflects changes in the condition of a system or process to convey details about the token's purpose, such as color function
+4. **Emphasis:** An optional term that helps to indicate where visual prominence is increased or decreased (e.g., bold, subtle)
+5. **Variant:** An optional term allowing variation within a set of attributes within a characteristic (e.g., primary, secondary, tertiary)
+6. **Property:** A mandatory term to indicate the UI attribute the token applies to (e.g., background, border, shadow, foreground)
+7. **State:** An optional term indicating the current condition of an element (e.g., hover, active, selected, disabled, read-only)
+
+If a token name doesn’t include a modifier, emphasis, variant or state, it represents the ‘default’ value. However, Salt avoids using the term ‘default’ in token names, as it can be ambiguous and doesn’t clearly describe the token’s purpose or usage. This approach helps keep token naming clear and specific.
+
+## Characteristic
+
+Semantic characteristics are groups of tokens that share the same purpose or intent. In Salt, every characteristic token includes its semantic group directly after the design system prefix, e.g., --salt-content-primary-foreground. Below are the semantic characteristics used in Salt:
```

Comment:
Semantic characteristics isn't a thing. "The characteristic layer contains groups of semantic tokens..."